ganga/GangaCore/Lib/Executable/Executable.py

```diff
@@ -288,3 +288,28 @@ def prepare(self, app, appconfig, appmasterconfig, jobmasterconfig):
 allHandlers.add('Executable', 'CREAM', LCGRTHandler)
 allHandlers.add('Executable', 'ARC', LCGRTHandler)
 allHandlers.add('Executable', 'Slurm', RTHandler)
+
+def randomString():
+    """Simple method to generate a random string"""
+    from random import randint
+    from string import ascii_uppercase
+
+    def addToSample(sample, ascii_length):
+        """Basically random.select but python2.2"""
+        a = ascii_uppercase[randint(0, ascii_length - 1)]
+        if not a in sample:
+            sample.append(a)
+        else:
+            # passing by referance
+            addToSample(sample, ascii_length)
+
+    ascii_length = len(ascii_uppercase)
+    sample = []
+    for _ in range(6):
+        addToSample(sample, ascii_length)
+    assert(len(sample) == 6)
+
+    # seed is set to clock during import
+    return ''.join([str(a) for a in sample])
+
+
```

Comment:
Out of context it seems strange that this was even in the ROOT module. Makes me wonder if it was only used in said module and maybe a candidate for removing altogether. Do you know where this is used?

Reply:
I'm referring to the `randomString` function which unfortunately didn't show here in it's entirety.

Reply:
Yes I agree it was weird. Would it be better put elsewhere?

Reply:
for now, putting it in `Executable` seems as good as anything. My point was more that if it was only used in the now defunct `Root` module then maybe we don't need it at all?

Reply:
Ah, It seems to be used in various places so I kept it.